README.md

```diff
@@ -12,36 +13,54 @@
-
 <!-- toc -->
 
-- [⚛️ Why React](#-why-react)
+# What's in this document
 
-- [🚀 Getting Started](#-getting-started)
+- [Introduction](#-introduction)
+- [Getting Started](#-getting-started)
+- [Supporting and contributing](#-supporting-and-contributing)
+- [Related Links](#-related-links)
 
-- [📌 Requirements](#-requirements)
+<!-- tocstop -->
 
-- [📚 Related Links](#-related-links)
+# 🐣 Introduction
 
-- [📣 Feedback](#-feedback)
+[Botonic](https://botonic.io) is a full-stack framework to create chatbots and modern [conversational apps](https://www.hubtype.com/blog/what-are-conversational-apps/).
 
-<!-- tocstop -->
+It's built on top of:
 
-# ️️️ ⚛️ Why React
+⚛️ [React](https://reactjs.org/)
 
-<p align="center">
-    <img
-      alt="Node.js"
-      src="https://botonic.io/img/botonic-logo.png"
-      width="350"
-    />
-</p>
+⚡ [Serverless](https://www.serverless.com/)
+
+💡 [Tensorflow.js](https://www.tensorflow.org/js)
+
+And it works on:
+
+💬 Messaging apps: **Whatsapp**, **Facebook Messenger**, **Telegram**, **Twitter DMs**
+
+🌐 Your website
+
+📱 Your mobile app
 
-Building chatbots is a lot like building websites, that's why Botonic uses the same abstractions used in React, like routes, components and JSX, so you can build modern conversational interfaces (with text, NLP, rich interactive messages and webviews) using technology you already know and love.
+With Botonic you can focus on creating the best conversational experience for your users instead of dealing with different messaging APIs, AI/NLP complexity or managing and scaling infrastructure.
+It also comes with a battery of plugins so you can easily integrate popular services into your project, for instance:
+
+- Analytics ([Google Analytics](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-google-analytics), [Segment](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-segment), [Dashbot](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-dashbot))
+- CMS ([Contentful](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-contentful), Sanity)
+- NLU ([Dialogflow](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-dialogflow), [Watson](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-watson), [Luis](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-luis), [Inbenta](https://github.com/hubtype/botonic/tree/master/packages/botonic-plugin-inbenta))
+
+_If you'd like to see more plugins/integrations please submit an issue or a pull request_
```

Comment:
_If you'd like to see more plugins/integrations, please submit an issue or a pull request_